docs/manual/install.md

```diff
@@ -50,3 +50,85 @@ $ mkdir build && cd build
 $ cmake -DCMAKE_INSTALL_PREFIX=<unique prefix> ..
 $ make install
 ```
+
+## Setting up Jupyter
+
+Since the latest remage versions, it is possible to include remage into your jupyter python kernel. This allows you to run everything within a single jupyter notebook. Because the setup is very easy, this might be more comfortable and easier than using the CLI (depending on your experience and use-case). Keep in mind that interactive visualization, like the pyg4ometry or Geant4 visualizer, might be very buggy in a notebook. You can still try it, but you can not say that we did not warn you!
+
+To set up your jupyter kernel, you only have to provide the correct `kernel.json` file to your jupyter installation.
+
+```console
+$ mkdir -p ~/.local/share/jupyter/kernels/remage
+$ touch ~/.local/share/jupyter/kernels/remage/kernel.json
+```
+
+:::{note} If you are using a jupyter that is installed inside of a virtual environment (or in vscode your default python interpreter is inside a venv) then you probably need to add the `kernel.json` to the share folder of that jupyter installation instead.
+:::
+
+If you have installed remage from a pre-built binary using apptainer, your kernel file should look something like this (remember to replace `/path/to/remage_latest.sif` with your actual path):
+
+```json
+{
+  "argv": [
+    "sh",
+    "-c",
+    "apptainer exec -B $XDG_RUNTIME_DIR:$XDG_RUNTIME_DIR /path/to/remage_latest.sif python -m ipykernel_launcher -f {connection_file}"
+  ],
+  "display_name": "remage container",
+  "language": "python",
+  "metadata": {
+    "debugger": true
+  }
+}
+```
+
+If you instead installed using docker it should look something like this:
+
+```json
+{
+  "argv": [
+    "sh",
+    "-c",
+    "docker run --rm --network=host --entrypoint python -v $XDG_RUNTIME_DIR:$XDG_RUNTIME_DIR -v $(pwd):$(pwd) -w $(pwd) legendexp/remage:dev -m ipykernel_launcher -f {connection_file}"
+  ],
+  "display_name": "remage (docker-dev)",
+  "language": "python",
+  "metadata": {
+    "debugger": true
+  }
+}
+```
+
+:::{note} On macOS `--network=host` is not supported. Try `"docker run --rm --entrypoint python -v $(pwd):/work -v $HOME/.jupyter/runtime:/root/.jupyter/runtime -w /work legendexp/remage:dev -m ipykernel_launcher -f {connection_file}"` instead. This is untested and not sure if it works.
+:::
```

Comment:
we cannot test this so I would just remove it. we can add a note in the future if anyone gives it a try on macos